### DISCUSSION TITLE
Lampadine BLE in Home Assistant

### DISCUSSION THREAD

**Discussion by NuccioT** [indomus/forum#2805] [Hardware & Firmware]
Salve a tutti e buon 2024!
Ho acquistato da Aliexpress delle lampadine smart che avrebbero dovuto essere zigbee ma sono invece BLE. Invece di restituirle le ho tenute volendo provare a implementare un ble gateway. Al proposito ho acquistato una scheda NodeMcu WT 32 ETH0.
Sono riuscito a installarla su ESPHome, risulta in linea ma non mi fa vedere neanche un dispositivo. Siccome sono nuovo al BLE vorrei sapere se per esempio il telefonino, l'echo show e altri dispositivi dovrebbero essere rilevati. Per quanto riguarda la lampadina funziona con l'app Smartlife ma siccome non gradisco condividere i miei dati con loro vorrei collegarla ad HA solo tramite BLE Gateway.
Se qualcuno ha voglia di approfondire posso pubblicare documentazione integrativa.
Grazie e buon anno a voi tutti

**Comment by 50494554524F** [comment]
se le lampadine usano smartlife sono tuya, puoi usare l'integrazione tuyalocal che trovi in hacs... probabilmente dovrai iscriverti a tuya developer per recuperare alcuni dati delle lampadine

per quanto riguarda esphome ti basta installare il bt gateway che trovi già pronto sul sito esphome oppure puoi creartelo "personalizzato" (è semplice) usando le varie integrazioni bluetooth di esphome

**Comment by NuccioT** [comment]
Grazie mille Pietro per i tuoi suggerimenti.
Il fatto è che non voglio avere nulla a che fare con tuya in quanto non mi fido molto.
Ho installato il BT gateway ma non mi rileva nulla: non so se dovrebbe rilevare per esempio il telefonino, echo show e altri dispositivi oppure se sia solo per devices tipo termostati, con pochi dati da trasmettere.
________________________________
Da: Pietro ***@***.***>
Inviato: giovedì 4 gennaio 2024 19:44
A: indomus/forum ***@***.***>
Cc: Nuccio ***@***.***>; Author ***@***.***>
Oggetto: Re: [indomus/forum] Lampadine BLE in Home Assistant (Discussion #2805)


se le lampadine usano smartlife sono tuya, puoi usare l'integrazione tuyalocal che trovi in hacs... probabilmente dovrai iscriverti a tuya developer per recuperare alcuni dati delle lampadine

per quanto riguarda esphome ti basta installare il bt gateway che trovi già pronto sul sito esphome oppure puoi creartelo "personalizzato" (è semplice)
